.github/workflows/CI.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
```

Comment:
[nits]
A more human readable name is considered desirable.
i.e. `build-and-test`

Reply:
It might be preferable if the file name were also.

Reply:
I was quite sure what to name the workflow on push should be since build-test-type-check seems too much, but I think ci is okay looking at other projects like vue. anyway I've renamed the workflow for publishing. Thanks for pointing  out 🙏 [415c57d](https://github.com/line/ts-remove-unused/pull/1/commits/415c57dd23517ec37d522a2c37baf55a4fb6bed3)